### ISSUE TITLE
Export a TSV file of disease associations

### ISSUE THREAD

**Issue by kimrutherford** [pombase/pombase-chado#888]
The PomBase curated input file (`pombase_disease_associations_mondo_ids.txt`) has six columns:

  - gene systematic ID
  - gene name
  - MONDO term ID
  - evidence code (always blank)
  - reference
  - date

I think it makes sense to use those columns except for the evidence code.

**Comment by kimrutherford** [comment]
Our nightly update now writes out a disease association file:
https://curation.pombase.org/dumps/latest_build/misc/disease_association.tsv

The columns are:

  - gene systematic ID
  - gene name
  - MONDO term ID
  - reference
  - date

Most of the references are PMIDs but some are "PB_REF:0000003" which is a PomBase internal reference.  It's defined here: https://curation.pombase.org/dumps/latest_build/pombe-embl/supporting_files/PB_references.txt
but I think it would be OK to just leave the reference in PombaseMine unset if it's "PB_REF:0000003" in the file. 

@rachellyne
@danielabutano

**Comment by ValWood** [comment]
Hi Kim,
Itwould be useful for me to take a copy of this and replace the old complicated pipelines that can now be deprracated. It will really streamline the editing and maintenance. We can chat about this.
v

**Comment by kimrutherford** [comment]
> Itwould be useful for me to take a copy of this and replace the old complicated pipelines that can now be deprracated. It will really streamline the editing and maintenance. 

One thing we get from the current system is a way to link to the Malacards pages.  When we load `malacards_data_for_chado_mondo_ids.tsv` we store the Malacards details with the MONDO terms.

**Comment by ValWood** [comment]
RIght. I don't think we will realign with  Malacards, they have tended to overpredict. In future I will check against 
Alliance (which covers lots of species/sources) and OMIM (via Intermine) and Monarch. So one option is just to take the current set of used MAlacards IDs in a MONDO mapping file . The part  which takes time at the moment is, that when something changes (MONDO obsolete terms  quite often) there are sometimes multiple places to track things down change things, and it always takes me a while to figure out where something is coming from. Especially as the malacards file is based on the human ID...it's all a bit convoluted.

It isn't urgent, but it would streamline the maintenance task. We can go through it next time a bunch of MONDO IDs are obsoleted.

**Comment by ValWood** [comment]
We can probably close this and open new tickets if anything else is needed. We have a separate issue about streamlining disease associations but that isn't urgent.
